File: pset1/mario.c
#include<cs50.h>
#include<stdio.h>
int main(void)
{
    int n;
    do
    {
     n = get_int("height:");
    }
    while(n<1 || n>8);
int y=0;
int x =n;
for(int i=0;i<=n;i++)
{
     for(int k=y;k<n;k++)
    {
        printf(" ");
    }
    for(int j=x;j<n;j=j+1)
    {
        printf("#");
    }

x=x-1;
y=y+1;
printf("\n");
}
}

File: sentimental-mario-more/mario.py
from cs50 import get_int
while(h<=0 or h>8):
  h=get_int("Height:")
a=1
b=h-1
for i in range(h):
    for j in range(b):
        print(' ',end='')

    for j in range(a):
        print('#',end='')
    print('  ',end='')
    for j in range(a):
        print('#',end='')
    a+=1
    b-=1
    print()
